ppl/src/main/java/org/opensearch/sql/ppl/parser/AstExpressionBuilder.java

```diff
@@ -332,8 +336,12 @@ private List<UnresolvedExpression> relevanceArguments(RelevanceExpressionContext
     // all the arguments are defaulted to string values
     // to skip environment resolving and function signature resolving
     ImmutableList.Builder<UnresolvedExpression> builder = ImmutableList.builder();
+    ParserRuleContext field = ctx.field;
+    if (field == null) {
```

Comment:
use conditions to define the ParseRuleContext, for example: 
ParserRuleContext field = ctx.field != null ? ctx.field : ctx.fields;